src/core/modes/walk-mode.ts

```diff
@@ -59,18 +59,33 @@ export class WalkMode implements Mode {
       return;
     }
 
-    const position = firstVisibleRange.start;
-    const range = new vscode.Range(position, position);
+    let visibleStartLine = firstVisibleRange.start;
+    let visibleEndLine = firstVisibleRange.start;
+    const junbaeLocation = { top: '5%', 'font-size': '50px' };
+    const location = vscode.workspace.getConfiguration('junbae-mode').get('location');
+
+    if (location === 'bottom') {
+      visibleStartLine = firstVisibleRange.end;
+      visibleEndLine = firstVisibleRange.end;
+      junbaeLocation.top = '-100px';
+      junbaeLocation['font-size'] = '100px';
+    }
+
+    const range = new vscode.Range(visibleStartLine, visibleEndLine);
 
     if (this.combo !== this.renderedComboCount || !range.isEqual(this.renderedRange!)) {
       this.renderedComboCount = this.combo;
       this.renderedRange = range;
-      this.createWalkMotionDecorator([range], editor);
+      this.createWalkMotionDecorator([range], editor, junbaeLocation);
       this.createTimerDecoration([range], editor);
     }
   }
 
-  private createWalkMotionDecorator = (ranges: vscode.Range[], editor: vscode.TextEditor) => {
+  private createWalkMotionDecorator = (
+    ranges: vscode.Range[],
+    editor: vscode.TextEditor,
+    junbaeLocation: { top: string; ['font-size']: string },
```

Comment:
I think we need about scalability or abstract... there are something else..

maybe we consider `Type` about location css of motions

Reply:
When creating a config module, it would be a good idea to consider scalability when added.